<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QWidget" name="dllWgt" native="true">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>0</y>
        <width>671</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="btnWgt" native="true">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>121</width>
        <height>601</height>
       </rect>
      </property>
      <widget class="QPushButton" name="visibleBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>270</y>
         <width>12</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:white;
	border: 1px solid rgb(120,200,255);
	color: rgb(120,200,255);
	border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
